torch/csrc/lazy/core/tensor_aten_ops.h

```diff
@@ -10,81 +10,81 @@ namespace lazy {
 //////////////////////////////////////////////////////////////////////////////
 // Takes a slice from the input as R1 at the specified offset and reshapes it
 // into the provided size.
-torch::lazy::LazyTensorPtr as_strided(const torch::lazy::LazyTensorPtr& input, std::vector<int64_t> size,
+TORCH_API torch::lazy::LazyTensorPtr as_strided(const torch::lazy::LazyTensorPtr& input, std::vector<int64_t> size,
```

Comment:
i'm actually not in favor of moving this file or exposing these APIs; I was secretly planning to delete these functions and just fold their implementations directly into ts_native_functions.cpp.  Are you planning to reuse these?

Reply:
I did move it with the intention of reusing them in the MLIR backend. However, if the plan is to remove them, then I'm fine with that. I can similarly make the change in the MLIR backend's equivalent to `ts_native_functions.cpp`. It might mean a bit more code is duplicated, but I'm all for making LTC simpler and cleaner overall.

Reply:
ok- makes sense, lets aim for the cleaned up version but if doing that later is easier i'm ok with that.